specification/machinelearningservices/AzureAI.Assets/tspconfig.yaml

```diff
@@ -3,6 +3,7 @@ parameters:
     default: "sdk/ai"
 emit:
   - '@azure-tools/typespec-autorest'
+  - "@azure-tools/typespec-python"
```

Comment:
Couldn't resolve import "@azure-tools/typespec-python" this causes compile error for me, did you do anything else to have this resolve?

Reply:
Yes. I updated `package.json` and run `tsp install`.